lampo-common/src/model/invoice.rs

```diff
@@ -54,7 +54,10 @@ pub mod response {
             Self {
                 bolt12: value.to_string(),
                 metadata: value.metadata().map(|bytes| hex::encode(bytes)),
-                metadata_pubkey: value.signing_pubkey().to_string(),
+                metadata_pubkey: value
+                    .signing_pubkey()
+                    .expect("Public key is missing")
```

Comment:
We do not use `expect` inside lampo, or we add a safety comment (look at some example with `git grep -n SAFETY`)

Expect is just an wrong suggestion of clippy

Reply:
So should I simply unwrap this and add a SAFETY comment?